tests/codeception/acceptance/LocalFooter/LocalFooterLockupCest.php

```diff
@@ -315,7 +337,9 @@ public function testLockupSettingsT(AcceptanceTester $I) {
     $I->fillField('Line 4', 'Organization name');
     $I->fillField('Line 5', 'Last line full width option');
     $I->click('Save');
+    $I->see('Local Footer has been updated');
     $I->amOnPage('/');
+    $I->runDrush('cr');
```

Comment:
thats alot of cache clearing. the cache clearing should occur in the form submit automatically. this smells like we should do something better with the form submit. `Cache::invalidateTags(['config:system.site']);` normally does the trick

Reply:
It should: https://github.com/SU-SWS/stanford_profile_helper/blob/8.x-1.x/stanford_profile_helper.module#L513

But something no worky in circly